dockerfiles/testsuite/genesis.json

```diff
@@ -4,7 +4,7 @@
   "initial_height": "0",
   "consensus_params": {
     "block": {
-      "max_bytes": "5242880",
+      "max_bytes": "2097152",
```

Comment:
Just a doubt, what does this limit mean and why is it being reduced?

Reply:
This is the size of the block. So the number of transactions that can be included in a period of time of 10-12 seconds.

I've performed several tests and benchmarking before creating this PR and 5 MiB seems too much. The block time easily increases to 30-40 seconds on high load (thousands of votes per minute continuously). However if using 2 MiB, the network performs stable, with a block time of 12-13 seconds on high load.